backend/templates/cms_base.html

```diff
@@ -0,0 +1,75 @@
+{% load cms_tags static sekizai_tags menu_tags %}
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+       <meta charset="utf-8"/>
+    <title>Blog</title>
+    <link rel="shortcut icon" href="data:image/x-icon;," type="image/x-icon">
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta name="theme-color" content="#000000"/>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+          rel="stylesheet"
+          integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+          crossorigin="anonymous">
+    <link
+        rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
```

Comment:
I don't believe you need most of these stylesheets adn scripts, definitely not leaflet (our mapping module)